src/components/TransactionForms/CustomForms/ConfigITO.tsx

```diff
@@ -183,7 +184,7 @@ export const WhitelistSection: React.FC = () => {
         <span>Whitelist</span>
       </SectionTitle>
       {fields.map((field, index) => (
-        <FormSection key={field.id} inner>
+        <FormSection2 key={field.id} inner style={{ top: '2.5rem' }}>
```

Comment:
Probably instead of creating a 'FormSection2' you can pass an optional prop 'paddingTop', like you did to the 'FormInput', and instead of boolean use the value. So we can use the same component with different values for the padding
Something  like:
`<FormSection key={field.id} inner paddingTop={ '2.5rem' } />`